ports/catch2/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "catch2",
-  "version-semver": "3.3.1",
+  "version": "3.3.2",
```

```suggestion
  "version-semver": "3.3.2",
```
I'm very sorry, I re-checked that the version control provided by the upstream code is version-semver, please restore the version control to version-semver, thank you.

---

No worries. :wink: